components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -139,22 +139,24 @@ function CustomInjector:parse(id, widgets)
 		--maps
 		if String.isNotEmpty(_args.map1) then
 			table.insert(widgets, Title{name = 'Maps'})
```

Comment:
Should this one have an overridable title? Seems useful is both map and 2map are present.

Reply:
good idea, added